eventing/README.md

```diff
@@ -230,6 +230,9 @@ FTP server for new files or generate events at a set time interval.
 - [Install Eventing components](#installation)
 - [Run samples](samples/)
 
+## Configuration
```

Comment:
It might be worth linking to this section at the end of the install section on this page.

Reply:
+1

Reply:
Done.